local-cli/generator/templates/index.android.js

```diff
@@ -15,11 +15,11 @@ import {
 export default class <%= name %> extends Component {
   render() {
     return (
-      <View style={styles.container}>
-        <Text style={styles.welcome}>
+      <View style={styles.container} testID="welcome_screen">
+        <Text style={styles.welcome} testID="welcome_header">
           Welcome to React Native!
         </Text>
-        <Text style={styles.instructions}>
+        <Text style={styles.instructions} testID="some_id_not_in_ids_xml">
```

Comment:
I don't think we should have "some_id_not_in_ids_xml" in every React Native app created by anyone. They might not be interested in e2e testing right away when trying out React Native for the first time.

Please leave out the testIDs from the template, it's discoverable enough in the docs.